stories/rating-indicator/rating-indicator.stories.js

```diff
@@ -0,0 +1,529 @@
+            <label class="fd-rating-indicator__label" for="rating-icon-2"></label>
+            
+            <input aria-label="3 star" type="radio" class="fd-rating-indicator__input" id="rating-icon-3" name="rating-icon" value="3">
+            <label class="fd-rating-indicator__label" for="rating-icon-3"></label>
+            
+            <input aria-label="4 star" type="radio" class="fd-rating-indicator__input" id="rating-icon-4" name="rating-icon" value="4">
+            <label class="fd-rating-indicator__label" for="rating-icon-4"></label>
+            
+            <input aria-label="5 star" type="radio" class="fd-rating-indicator__input" id="rating-icon-5" name="rating-icon" value="5">
+            <label class="fd-rating-indicator__label" for="rating-icon-5"></label>
+        </div>
+        <span class="fd-rating-indicator__dynamic-text">(2 of 5)</span>
+    </div>
+</div>
+`;
+
+CustomIcons.parameters = {
+    docs: {
+        storyDescription: `
+To make the rating indicator to use custom icons needs to be added class \`.fd-rating-indicator--icon\` 
+to the \`fd-rating-indicator\` element.
+Additionally, needs to set css variables \`--sapRating-indicator-icon-rated: url(...)\` and \`--sapRating-indicator-icon-unrated: url(...)\`
+`
+    }
+};
+
+export const ReadOnly = () => `
+<div class="example-container">
+    <div class="fd-rating-indicator" aria-readonly="true">
+        <div class="fd-rating-indicator__container" aria-label="Star Rating (out of 5)">
+            <input aria-label="1 star" type="radio" class="fd-rating-indicator__input" id="rating-readonly-1" name="rating-readonly" value="1" readonly>  
+            <label class="fd-rating-indicator__label" for="rating-readonly-1"></label>
+            
+            <input aria-label="2 star" type="radio" class="fd-rating-indicator__input" id="rating-readonly-2" name="rating-readonly" value="2" readonly checked>
+            <label class="fd-rating-indicator__label" for="rating-readonly-2"></label>
+            
+            <input aria-label="3 star" type="radio" class="fd-rating-indicator__input" id="rating-readonly-3" name="rating-readonly" value="3" readonly>
+            <label class="fd-rating-indicator__label" for="rating-readonly-3"></label>
+            
+            <input aria-label="4 star" type="radio" class="fd-rating-indicator__input" id="rating-readonly-4" name="rating-readonly" value=4" readonly>
+            <label class="fd-rating-indicator__label" for="rating-readonly-4"></label>
+            
+            <input aria-label="5 star" type="radio" class="fd-rating-indicator__input" id="rating-readonly-5" name="rating-readonly" value="5" readonly>
+            <label class="fd-rating-indicator__label" for="rating-readonly-5"></label>
+        </div>
+        <span class="fd-rating-indicator__dynamic-text">(2 of 5)</span>
+    </div>
+</div>
+`;
+
+ReadOnly.parameters = {
```

Comment:
Change name to "Read-Only" for consistency.

Reply:
fixed